IFC4x3/Properties/h/HydraulicConductivity_19kpZnztn1VQFucvmNUC96/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="HydraulicConductivity_19kpZnztn1VQFucvmNUC96" Name="HydraulicConductivity" UniqueId="49bb38f1-f77c-417d-a3f8-9b9c1778c246" PrimaryDataType="IfcVelocityMeasure" />
```

Comment:
Not too happy about `IfcVelocityMeasure` here. How about `IfcVaporPermeabilityMeasure`? Not the same but at least it is about water/vapor permeability.

Reply:
> Not too happy about `IfcVelocityMeasure` here. How about `IfcVaporPermeabilityMeasure`? Not the same but at least it is about water/vapor permeability.

The definition mentions "given with the K or Kf value in **m/s**". Vapor permeability seems to be measured in kg / (s * m2 * Pa). @j-weil

Reply:
Yes, even if units are the same, the measure is not. It is measured differently than linear velocity.

Reply:
Keep velocity according to domain experts.